api/export_api.py

```
@@ -208,7 +216,10 @@ def export_chats():
 
     except Exception as e:
-        print(f"Export error: {e}")
-        import traceback
-        traceback.print_exc()
+        _logger.error(
+            "Export failed: %s (%s)",
+            e,
+            type(e).__name__,
+            exc_info=True,
+        )
         return jsonify({"error": f"Export failed: {str(e)}"}), 500
```

Ruff (0.15.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)